packages/components/src/experience-control/index.stories.tsx

```diff
@@ -1,8 +1,9 @@
 import { Meta, StoryObj } from '@storybook/react';
-import { ExperienceControl } from './index';
+import { fn } from '@storybook/test';
+import { ExperienceControl, ExperienceType } from './index';
 
 const meta: Meta< typeof ExperienceControl > = {
-	title: 'Components/ExperienceControl',
+	title: 'packages/components/ExperienceControl',
 	component: ExperienceControl,
```

Comment:
```suggestion
	component: ExperienceControl,
	args: { onChange: fn() },
```

FYI we can set all the `onChange` spies in the `meta`.